ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -150,6 +150,11 @@ spec:
     - name: ssh-dir
       optional: true
     - name: registry-credentials
+    - name: registry-pull-credentials
```

Comment:
No need to add another workspace. The `hosted-pipeline-registry-auth-secret` should already cover this.